include/boost/gil/image_processing/harris.hpp

```diff
@@ -0,0 +1,74 @@
+#ifndef BOOST_GIL_HARRIS_HPP
+#define BOOST_GIL_HARRIS_HPP
+
+#include <boost/gil/image_view.hpp>
+#include <boost/gil/typedefs.hpp>
+
+namespace boost { namespace gil {
+/// \defgroup CornerDetectionAlgorithms
+/// \brief Algorithms that are used to find corners in an image
+///
+/// These algorithms are used to find spots from which
+/// sliding the window will produce large intensity change
+
+
+/// \brief function to record Harris responses
+/// \ingroup CornerDetectionAlgorithms
+///
+/// This algorithm computes Harris responses
+/// for structure tensor represented by m11, m12_21, m22 views.
+/// Note that m12_21 represents both entries (1, 2) and (2, 1).
+/// Window length represents size of a window which is slided around
+/// to compute sum of corresponding entries. k is a discrimination
+/// constant against edges (usually in range 0.04 to 0.06).
+/// harris_response is an out parameter that will contain the Harris responses.
+void compute_harris_responses(
+    gil::gray32f_view_t m11,
+    gil::gray32f_view_t m12_21,
```

Comment:
This namespace qualification is confusing a bit. It's referring to current scope namespace already, no?
I think it's not necessary.

Reply:
It is used in filter generation, and it is used throughout the file. I can add this as issue so we could review this after GSoC is done, what do you think?

Reply:
I made qualification consistent, now both have full qualification. I'll add the question as issue, so we could review this after GSoC is over and include into file cleanup that you mentioned.

Reply:
Looks good, thanks for follow-up in #352